Formula/emqx.rb

```diff
@@ -0,0 +1,69 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.9.tar.gz"
+  sha256 "bc5d193a02840b1d4cb5f6c9f047003b442c5a0b4a3ccb1cfeedafacf04ff82e"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  def dashboard_version
+    "1.1.0"
+  end
+
+  def emqx_version
+    "5.0.9"
+  end
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
+  depends_on "coreutils" => :build
+  depends_on "erlang@24" => :build
+  depends_on "freetds"   => :build
+  depends_on "libtool"   => :build
+  depends_on "openssl@1.1"
```

Comment:
Does this not work with the most recent versions?

Reply:
No, Erlang/OTP only added partial support for openssl v3 in 25.1 release, and we haven't adopted it yet.

Reply:
New formulae should not be using outdated dependencies

Reply:
does it mean that the formula will not be accepted as long as we depend on openssl@1.1?

Reply:
The linkage check already shows it depends on openssl 3.

Reply:
my information was wrong
- erlang@24, in fact, depends on openssl@3
- we don't need to explicitly list openssl as a dependency
fixed.